code/modules/mob/living/simple_animal/hostile/ordeal/indigo.dm

```diff
@@ -349,14 +349,28 @@
 /mob/living/simple_animal/hostile/ordeal/indigo_midnight/proc/ValueTarget(atom/target_thing, hate_value = 0)
 	if(isliving(target_thing))
 		var/mob/living/L = target_thing
+
+		//Minimum starting hate for anything living is 80.
+		hate_value += 80
+		//Hate for corpses since we eats them.
 		if(L.stat == DEAD)
-			return 100
-		hate_value += 1
-		if(iscarbon(L))
 			hate_value += 10
-		if(L.health <= (L.maxHealth * 0.6))
-			hate_value += (10 * ( L.health / L.maxHealth ))
+		//Highest possible addition is + 9.9
+		if(iscarbon(L))
+			if(L.stat != DEAD && L.health <= (L.maxHealth * 0.6))
+				hate_value += (10 * ( L.health / L.maxHealth ))
```

Comment:
wait, isn't this math wrong?
This means the lower health you get, the lower your "hate value".
Having 100 health out of 100 health means you get 10x1, so +10.
Having 50 out of 100 health means you get 10x0.5, so +5.
```suggestion
				var/upper = L.maxHealth - HEALTH_THRESHOLD_DEAD
				var/lower = L.health - HEALTH_THRESHOLD_DEAD
				hate_value += min( 2 * ( 1 / ( max( lower, 1 ) / upper ) ) ), 20)
```
This serves to include the death threshold as well as making it now a range of ~3.3 (when it starts) to 20 (when it caps at 10% hp)

Reply:
No your right higher health would be prioritized the way i used to do it.